python_only_dev.py

```diff
@@ -69,7 +69,8 @@
     current_vllm_path = os.path.join(cwd, "vllm")
 
     print(f"Renaming {pre_built_vllm_path} to {tmp_path} for backup")
-    os.rename(pre_built_vllm_path, tmp_path)
```

Comment:
is it mentioned anywhere that this does not support rename between different devices in linux?

Reply:
https://docs.python.org/3/library/os.html#os.rename

> On Windows, if dst exists a [FileExistsError](https://docs.python.org/3/library/exceptions.html#FileExistsError) is always raised. The operation may fail if src and dst are on different filesystems. Use [shutil.move()](https://docs.python.org/3/library/shutil.html#shutil.move) to support moves to a different filesystem.